package.json

```diff
@@ -156,26 +160,55 @@
     "node-abi": "3.28.0",
     "node-gyp": "9.3.0",
     "node-watch": "0.7.3",
-    "parcel-bundler": "1.12.5",
+    "parcel": "2.8.0",
     "prebuild-install": "7.1.1",
+    "punycode": "1.4.1",
+    "querystring-es3": "0.2.1",
     "rimraf": "3.0.2",
     "shx": "0.3.4",
     "standardx": "7.0.0",
+    "stream-browserify": "3.0.0",
```

Comment:
couple new deps here. do we know what they do and/or know that they're all needed? just want to keep them as much to a minimum as possible

Reply:
Yeah, Parcel installs node polyfills